LICENSES.TXT

```diff
@@ -0,0 +1,30 @@
+The contents of this repository are subject to the RDK License Agreement.  Please refer to the RDK License Agreement signed by your company.
+
+The component may include material which is licensed under other licenses / copyrights as
+listed below. Your use of this material within the component is also subject to the terms and
+conditions of these licenses.
+
+The essos-test directory tree within this component contains some software which is
+Copyright 2017 RDK Management, Copyright 2020 Liberty Global B.V. and licensed to you
+under the Apache 2.0 License.  The essos-test directory contains its own license files,
```

Comment:
>The essos-test directory contains its own license files,
called LICENSE/NOTICE,

There is no such a file(s).

Reply:
these files are added in this PR:
https://github.com/stagingrdkm/dac-examples-src/pull/4/